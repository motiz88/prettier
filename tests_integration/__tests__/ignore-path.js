"use strict";

const runPrettier = require("../runPrettier");

test("ignore path", () => {
  const result = runPrettier("cli/ignore-path", [
    "**/*.js",
    "--ignore-path",
    ".gitignore",
    "-l"
  ]);

  expect(result.stdout).toMatchSnapshot();
  expect(result.status).toEqual(1);
});

test("support .prettierignore", () => {
  const result = runPrettier("cli/ignore-path", ["**/*.js", "-l"]);

  expect(result.stdout).toMatchSnapshot();
  expect(result.status).toEqual(1);
});
