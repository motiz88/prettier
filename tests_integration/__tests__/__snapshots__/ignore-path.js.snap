// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ignore path 1`] = `
"regular-module.js
"
`;

exports[`support .prettierignore 1`] = `
"other-regular-modules.js
"
`;
